libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -827,6 +827,15 @@ float GCS_MAVLINK::telemetry_radio_rssi()
     return last_radio_status.rssi/254.0f;
 }
 
+bool GCS_MAVLINK::last_txbuf_is_greater(uint8_t txbuf_limit)
+{
+    if (AP_HAL::millis() - last_radio_status.received_ms > 5000) {
```

Comment:
```suggestion
    if ((last_radio_status.received_ms == 0) || ((AP_HAL::millis() - last_radio_status.received_ms) > 5000)) {
```
Check for zero to make sure we do the correct thing if never received.

Reply:
Would a comment pointing out the the initialization of txbuf accomplishes this be OK instead?

Reply:
Yeah, comment is OK too. Really the key thing is to be able to understand what is going on without having to go look in the `.h`.